game.py

```diff
@@ -30,22 +29,28 @@ def user_name_answer():
 
 
 def create_score_file(name, score, score_file):
-    with open(score_file, 'w') as score_file, open('results_dict.pkl', 'wb') as pickle_file:
+    with open(score_file, 'w') as score_file:
         writer = csv.writer(score_file)
         writer.writerow(COLUMN_NAMES)
         writer.writerow([name, score])
-        results_dict = {name: score}
-        pickle.dump(results_dict, pickle_file)
 
 
-def rewrite_user_score(name, score, score_file):
-    with open(f'{score_file}') as old_file, open('temp_file.csv', 'w') as temp_file:
+def filling_score_file (name, score, score_file):
```

Comment:
your names of the files are always the same - `Score.csv` and `temp_file.csv`. Maybe it makes sense to write them once somewhere? That way you won't need to pass file names as parameters